database.sql

```diff
@@ -0,0 +1,94 @@
+/*To create this database locally in your system  follow the following steps
```

Comment:
This file is present in database folder also right?If yes, you can remove this file